core/class_db.h

```diff
@@ -36,7 +36,7 @@
 #include "core/print_string.h"
 
 /** To bind more then 6 parameters include this:
- *  #include "core/method_bind_ext.gen.inc"
+ *
  */
```

Comment:
Does it mean that it's not required to include `core/method_bind_ext.gen.inc` to have more than 5 parameters while binding methods anymore? See godotengine/godot-proposals#1229. If yes, then there are several places where this limit is advertised in the documentation as well, such as [Custom modules](https://docs.godotengine.org/en/latest/development/cpp/custom_modules_in_cpp.html).

Reply:
No. This is no longer required as everything uses variadic templates. Feel free to PR changes to the docs.